packages/nimble-components/src/dialog/tests/dialog-matrix.stories.ts

```diff
@@ -60,3 +103,48 @@ export const dialogDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
 );
 
 dialogDarkThemeBlackBackground.play = playFunction;
+
+export const smallDialogSizing: StoryFn = createStory(html`
+    ${createMatrix(dialogSizingTestCase, [
+        [
+            [
+                `Width ${dialogSmallWidth.getValueFor(document.body)}`,
+                `width: ${dialogSmallWidth.getValueFor(document.body)}`
+            ]
+        ],
+        [['Height fit-content', 'height: fit-content']],
+        [
+            [
+                `Max height ${dialogSmallMaxHeight.getValueFor(document.body)}`,
+                `max-height: ${dialogSmallMaxHeight.getValueFor(document.body)}`
+            ]
+        ]
+    ])}
+`);
+
+smallDialogSizing.play = playFunction;
+
+export const largeDialogSizing: StoryFn = createStory(html`
+    ${createMatrix(dialogSizingTestCase, [
+        [
+            [
+                `Width ${dialogLargeWidth.getValueFor(document.body)}`,
+                `width: ${dialogLargeWidth.getValueFor(document.body)}`
```

Comment:
Same as below, for all these use the css property name and don't worry about calculating the value

```suggestion
                `Width Large Dialog`,
                `width: var(${dialogLargeWidth.cssCustomProperty})`
```